limitador-server/src/http_api/server.rs

```diff
@@ -61,8 +74,8 @@ impl ResponseError for ErrorResponse {
 
 // Used for health checks
 #[api_v2_operation]
-async fn status() -> web::Json<()> {
-    Json(())
+async fn status(data: web::Data<RateLimitData>) -> web::Json<Status> {
+    Json(data.get_ref().status())
```

Comment:
I am assuming Json() can handle the parameter being &Status also.

Reply:
Possibly, but see [above](https://github.com/Kuadrant/limitador/pull/412#discussion_r1935550205).